pom.xml

```diff
@@ -79,11 +79,13 @@
 		<jacksonVersion>1.8.3</jacksonVersion>
 		<junitParamsVersion>1.0.4</junitParamsVersion>
 		<legacyuiVersion>1.4.0</legacyuiVersion>
+		<sync2Version>1.6.0</sync2Version>
+		<atomfeedVersion>1.6.0</atomfeedVersion>
```

Comment:
@echiteri Atom feed does not have a 1.6.0 version

Reply:
@ssmusoke I updated this